sdk/src/loom/System.ls

```diff
@@ -0,0 +1,131 @@
+package loom
+{
+    import system.platform.Platform;
+    import system.platform.PlatformType;
+    
+    public class System
```

Comment:
Does this need to wrap a natSystem or can it just BE a natSystem with extra LoomScript goodies like String?